src/style.css

```diff
@@ -1,3 +1,73 @@
+@font-face {
+  font-family: Roboto;
+  src: url(font/Roboto-Regular.ttf);
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(font/Roboto-Medium.ttf);
+  font-weight: 500;
+}
+
 body {
+  --color-active: #00acdc;
+
+  margin: 0;
+  background-color: #ddd;
+}
+
+header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
   margin: 0;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  position: relative;
+  text-decoration: none;
+  font-weight: 500;
+  color: #000;
+  font-size: 12px;
+  font-family: Roboto, Helvetica, sans-serif;
+  height: 60px;
+  display: flex;
+  align-items: center;
+}
+
+.logo__link {
+  height: 40px;
+}
+
+.nav__item:hover {
+  color: var(--color-active);
+}
+
+.is-active {
+  color: var(--color-active);
+}
+
+a.is-active::after {
+  position: absolute;
+  top: 56px;
+  right: 0;
+  content: '';
+  display: block;
+  background-color: var(--color-active);
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+}
+
+.list__item:not(:first-child) {
+  margin-left: 20px;
 }
```

Comment:
The selector '.list__item:not(:first-child)' seems to be applying a left margin to list items, but the checklist advises to be consistent with vertical margins and to add only top or only bottom margin, but not both. Although this is a horizontal margin rule, you might want to ensure that your margin usage is consistent throughout your CSS to avoid unexpected layout shifts.